app/assets/stylesheets/layouts/header.scss

```diff
@@ -0,0 +1,28 @@
+.page-header{
+  height: 60px;
+  background-color: #f3c937;
```

Comment:
Is this a brand color? Should we set this to a variable? `$primary-yellow-color: #f3c937;`